ui/tests/acceptance/oidc-config/scopes-test.js

```diff
@@ -9,8 +9,18 @@ import logout from 'vault/tests/pages/logout';
 import { create } from 'ember-cli-page-object';
 import fm from 'vault/tests/pages/components/flash-message';
 const flashMessage = create(fm);
+import {
+  clearRecord,
+  overrideMirageResponse,
+  SCOPE_LIST_RESPONSE,
+  SCOPE_DATA_RESPONSE,
+  PROVIDER_LIST_RESPONSE,
+  PROVIDER_DATA_RESPONSE,
+} from '../../helpers/oidc-config';
```

Comment:
Just saw this in Claire's PR, but it looks like this might be an easier path to import from `import { overrideMirageResponse, CLIENT_LIST_RESPONSE } from 'vault/tests/helpers/oidc-config';`